optimum/amd/ryzenai/quantization.py

```diff
@@ -163,19 +163,40 @@ def quantize(
         quantized_model_path = save_dir.joinpath(f"{self.onnx_model_path.stem}{suffix}").with_suffix(".onnx")
 
         LOGGER.info("Quantizing model...")
+
+        format = quantization_config.map_format().value
+        calibration_method = quantization_config.map_calibration_method().value
+        activation_type = quantization_config.map_activations_dtype().value
+        weight_type = quantization_config.map_weights_dtype().value
```

Comment:
I don't understand the need for this level of complexity, why need a `map_xxx().value`? Why not `format = quantization_config.format`?

Reply:
Updated